### ISSUE TITLE
Video is getting paused when switching from fullscreen to normal mode on iOS 11

### ISSUE THREAD

**Issue by akamdar** [videojs/video.js#5325]
## Description
I just encounter this issue only on iOS 11. Video is getting paused when switching from Fullscreen mode to Normal mode. I have also tried to play video from https://videojs.com/advanced/ link and it is also happening on it.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Play a Video
2. Switch to fullscreen mode
3. Now again switch to normal mode and you will see that video is getting paused.

### Expected
If video is playing in fullscreen mode then it should continue playing after exiting from fullscreen mode.

### Actual
Video is getting paused when exiting from fullscreen mode to normal mode.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs (All version)
what version of videojs does this occur with?
#### iOS Safari
what browser are affected?
#### iOS 11
what platforms (operating systems and devices) are affected?

**Comment by ldayananda** [comment]
Thanks for submitting an issue! If I recall correctly the native Safari player is used when switching into fullscreen mode on iOS. Would you be able to test if the same behavior is seen when using a video element without video.js as well?

**Comment by arski** [comment]
Confirmed that it does happen with a plain <video tag and without video.js, so probably not a video.js specific bug

**Comment by zhangciwu** [comment]
Got this problem using `video.js`, while `Dplayer` has no problem after exit fullscreen

**Comment by zhangciwu** [comment]
https://github.com/videojs/video.js/blob/2e6199074dc77259f45943e5f7bce4d65f70931d/src/js/control-bar/play-toggle.js#L58

modify this function into:
```
  handleClick(event) {
    if (this.player_.paused()) {
      this.player_.play();
      this.handlePlay();
    } else {
      this.player_.pause();
      this.handlePause();
    }
  }
```

make it better, after exit fullscreen, second click pause button will change button into play, but not perfect, it should be play button after exit fullscreen and got paused

**Comment by zhangciwu** [comment]
A thought: after exit fullscreen in iOS, the event listener got broken and thus can not respond to play/pause change

**Comment by gkatsev** [comment]
We have a potential fix to iphone having issues coming out of fullscreen with new versions of Video.js. See #6735.

**Comment by zhangciwu** [comment]
@gkatsev Tried with master with this PR merged, seems still broken
Codepen: https://codepen.io/zhangciwu/pen/BajrEyL
https://cdpn.io/zhangciwu/fullpage/BajrEyL

**Comment by gkatsev** [comment]
Oh, I misread the issue and just assumed it was related to the fix we just merged.
On iOS when you exist fullscreen the video will pause unless you exit by pinching the video closed. This is a platform limitation. I just verified with a plain video element.

**Comment by zhangciwu** [comment]
@gkatsev This is fine, but after exit of fullscreen, the button stuck at "pause button", no matter how many times you click it (the video would be controlled to play of pause, but the button just won't change)

**Comment by gkatsev** [comment]
@zhangciwu Your example does not contain the fix that we just merged (https://github.com/videojs/video.js/pull/6735) that fixes that issue.

**Comment by zhangciwu** [comment]
@gkatsev Found that now it's `main` branch than `master`,
with `main` branch code, it's working fine now. [(codepen)](https://codepen.io/zhangciwu/pen/JjGvrYY)

**Comment by gkatsev** [comment]
I also released 7.8.4 today with the fix.
